style/page.scss

```diff
@@ -0,0 +1,17 @@
+// This file is only to be used for brining in styles that allow us ot test
+// this component during development. Any styles that should be exported
+// along with the component go in style/main.scss.
+
+@import '../bower_components/o-colors/main';
```

Comment:
Could we remove the `../bower_components/` bit now? It seems you got it working.